docker/grafana/provisioning/datasources/influxdb-iml-datasource.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: 1
```

Comment:
How come this wasn't needed for RPM based installs?

Reply:
Because it's applied in the service config: https://github.com/whamcloud/integrated-manager-for-lustre/blob/master/chroma_core/lib/service_config.py#L336-L373

For the docker container we want to put the config in the provisioning folder such that when the container loads it will automatically configure the datasources.

Reply:
Interesting.... How come we don't want to do the same in the RPM based config then? That seems more straightforward then what we're currently doing with an API call, and will dedup the config, which is my main concern here.

Reply:
I agree that it's a cleaner approach. I think when the rpm was being setup we needed an entry point to do some setup and so it was added to the service config. We could probably take a similar approach though as it seems much cleaner.